src/tools/tidy/src/features.rs

```diff
@@ -7,13 +7,22 @@
 //! * Library features have at most one stability level.
 //! * Library features have at most one `since` value.
 //! * All unstable lang features have tests to ensure they are actually unstable.
+//! * Language features in a group are sorted by `since` value.
 
 use std::collections::HashMap;
 use std::fmt;
 use std::fs::{self, File};
 use std::io::prelude::*;
 use std::path::Path;
 
+use regex::{Regex, escape};
+
+mod version;
+use self::version::Version;
+
+const FEATURE_GROUP_START_PREFIX: &str = "// feature-group-start:";
+const FEATURE_GROUP_END_PREFIX: &str = "// feature-group-end";
```

Comment:
```suggestion
const FEATURE_GROUP_END_PREFIX: &str = "// feature-group-end:";
```

Reply:
I prefer just not requiring the colon for either as we don't really care about the group name other than as a scoping mechanism.